src/main/java/uk/gov/hmcts/reform/preapi/media/MediaKind.java

```diff
@@ -211,7 +232,7 @@ private void stopAndDeleteLiveEvent(String liveEventName) {
     private void stopAndDeleteStreamingEndpoint(String endpointName) {
         try {
             mediaKindClient.stopStreamingEndpoint(endpointName);
-        } catch (NotFoundException e) {
+        } catch (FeignException.NotFound e) {
             // ignore
             return;
```

Comment:
```suggestion
        } catch (NotFoundException e) {
            // ignore
            return;
```

Reply:
I don't think the NotFoundException is correct here. It's catching exceptions thrown by the Feign client so it'll throw a FeignException.NotFound not a NotFoundException which is one of ours?

Reply:
you're right, I even wrote a unit test for it before - thanks